src/consumer/component-ops/dependency-status.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import * as packageJson from '../component/package-json';
+export type DependencyStatusResult = { missing_files: string[] };
+//import { DEFAULT_BINDINGS_PREFIX } from '../../../src/constants';
+export type DependencyStatusProps = { main_file: string };
+
+async function getTopLevelDependencies(consumer: Consumer, dependencyStatusProps:DependencyStatusProps) {
+    const driver = await consumer.driver.getDriver(false);
+    let files = [];
+    files.push(dependencyStatusProps.main_file);
+    const consumerPath = consumer.getPath();
+    const tree = await driver.getDependencyTree(consumerPath, consumerPath, files, '@bit');
+    const top_level_dependencies = Object.keys(tree.tree).map(top_level_file=>top_level_file);    
```

Comment:
- [ ] linting (space around => )

- [ ] camel case

- [ ] why do you need the map here?